src/components/dungeonView.html

```diff
@@ -119,7 +119,8 @@ <h2 class="pageItemFooter" style="display: block; font-size: 1rem;">
 
                   <!--Pokérus image-->
                   <knockout data-bind="if: RouteHelper.minPokerusCheck(DungeonRunner.dungeon.allAvailablePokemon())">
-                      <img data-bind="attr:{ src: 'assets/images/breeding/pokerus/' + GameConstants.Pokerus[RouteHelper.minPokerus(DungeonRunner.dungeon.allAvailablePokemon())] + '.png'}"
+                      <img data-bind="attr:{ src: 'assets/images/breeding/pokerus/' + GameConstants.Pokerus[RouteHelper.minPokerus(DungeonRunner.dungeon.allAvailablePokemon())] + '.png',
+                            title: RouteHelper.dungeonPokerusEVs(DungeonRunner.dungeon)}"
```

Comment:
Use the tooltip attribute we are using other places

Reply:
Tooltip is possible only in Button, IMG uses title, same as images above (pokeballs)